src/pd/client.rs

```diff
@@ -299,8 +299,8 @@ impl PdClient for RpcClient {
         req.set_keys_written(region_stat.written_keys);
         req.set_bytes_read(region_stat.read_bytes);
         req.set_keys_read(region_stat.read_bytes);
-        req.set_approximate_size(region_stat.approximate_stat.size);
-        req.set_approximate_rows(region_stat.approximate_stat.rows);
+        req.set_approximate_size(region_stat.approximate_size);
+        req.set_approximate_write_keys(region_stat.approximate_write_keys);
```

Comment:
What does `write keys` mean?

Reply:
means the number of keys in the write cf